pkgs/games/freedroidrpg/default.nix

```diff
@@ -18,6 +18,8 @@ in stdenv.mkDerivation {
       url = "https://gitlab.com/freedroid/freedroid-src/-/commit/e610d427374226b79da5258d979936459f30c761.patch";
       sha256 = "1s7sw4dkc7b6i72j6x47driq6v0k3wss48l9ivd4fw40n3iaxjb1";
     })
+
+    ./drop-build-deps.patch
```

Comment:
Would be nice to add a short explanatory comment here so that maintainers do not have to open the file to find what it does. Something like “Do not embed build flags in the binary to reduce closure size.”

Reply:
Sounds good! Added comment you suggested as is.